frontend/src/metabase/query_builder/components/expressions/ExpressionEditorTextfield.jsx

```diff
@@ -147,6 +147,10 @@ export default class ExpressionEditorTextfield extends React.Component {
     if (suggestion) {
       const { tokens } = tokenize(source);
       const token = tokens.find(t => t.end >= suggestion.index);
+
+      const { editor } = this.input.current;
```

Comment:
Just to confirm, by the time we reach here, `this.input.current` can't be `null`, can it?

Reply:
@ariya I believe it can't be `null` because, for a suggestion to have been selected (this is all inside function `onSuggestionSelected`), the editor has to have been instantiated first.